doc/backing-store-crd.md

```diff
@@ -135,23 +141,26 @@ metadata:
   finalizers:
   - noobaa.io/finalizer
   labels:
-    app: noobaa
-  name: bs
-  namespace: noobaa
+    app: <>
+  name: <>
+  namespace: <>
 spec:
   azureBlob:
     secret:
-      name: backing-store-azure-blob-bs
-      namespace: noobaa
-    targetBlobContainer: CONTAINER
+      name: <>
+      namespace: <>
+    targetBlobContainer: <>
   type: azure-blob
 ```
 
-#### PV-POOL type
+## PV-POOL type
+Creataes a StatefulSet with a PVC mounted in each pod. Each resource will connect to the NooBaa core and provide the PV filesystem storage to be used for storing encrypted chunks of data. It is possible to configure the number of pods to be used and their PV size.
+
+Each PV needs at least 17 Gibibytes to function properly.
```

Comment:
Where is it defined in the code?

Reply:
I would write it as a recommendation (and not decide for the user on the buffer).
`It is recommended to set each PV with at least 17 Gibibytes (16 Gibibytes is the minimum that was defined in the system).`